### DISCUSSION TITLE
mae - .vcf filtering

### DISCUSSION THREAD

**Discussion by bpuzek** [gagneurlab/drop#397] [Q&A] [answered]
Hi team DROP, thanks for the great tool and the ongoing support!

I want to run MAE module and wanted to check in with you about the preprocessing of the .vcf file.

If I understand correctly from the docs, `qcVcf` will remove any variants that are not in the 1kg variant file.
After the analysis, the results are filtered for MAF based on the values of `maxAF`, `maxVarFreqCohort`.

My questions are:
1. Do you recommend any other filtering of the .vcfs upstream from MAE (such as VQSR)? If yes, would you recommend being more stringent or more lenient with the filter?
2. Would it be beneficial to do MAF filtering _before_ MAE runs, to reduce computation? 

Thanks a lot in advance,
Barbara

**Comment by vyepez88** [comment, marked as answer]
Hi Barbara,
Thanks for using DROP and your enquiry. 
The MAE module is composed of 2 parts, the test for MAE and the sampleQC. 
The sampleQC modules uses the 1kg variant file to detect overlapping variants in a sample's DNA and RNA.

Regarding, the test for MAE, a results table is exported per sample containing all tested SNVs located under `{root_dir}/processed_results/mae/samples/{vcf}--{rna}_res.Rds`. If `addAF: TRUE`, columns containing the gnomAD `maxAF` are included, as well as another column called `rare` which is `TRUE/FALSE` depending on the `maxAF` cutoff provided. 

We further export tables containing the MAE results of all samples together. Here, results are filtered as well by significance and rare.
In this case for the `rare` column is updated with the frequency of the variant in the whole cohort.

1. I usually recommend being more lenient as we're interested in finding genes or variants related to disease.
2. The computation time of the MAE test is negligible. Creating the allelic counts does take a few hours/sample. However, I would not recommend saving computational time at the risk of losing valuable variants.

**Reply by bpuzek** [reply]
Great, thank you!
